src/components/FeaturedSlider/styles.scss

```diff
@@ -0,0 +1,50 @@
+.home--slider {
```

Comment:
Notice that when you click the prev or next button, there is some weird effect that affects slider container height while img loads, you may preset a min height so there is no such effect, unless this is a wanted effect :)